snowpack.config.js

```diff
@@ -1,38 +0,0 @@
-/** @type {import("snowpack").SnowpackUserConfig } */
-module.exports = {
-  env: {
-    RADICLE_ALCHEMY_API_KEY: process.env.RADICLE_ALCHEMY_API_KEY,
-    RADICLE_HTTP_API: process.env.RADICLE_HTTP_API,
```

Comment:
This works out of the box with vite?

Reply:
Yeah I think I have to add this to the env variables in vite.config

Reply:
Vite works with dotenv loading all the env vars that are located in a .env file in the root directory.
https://vitejs.dev/guide/env-and-mode.html#env-files